streaming/base/converters/dataframe_to_mds.py

```diff
@@ -201,12 +160,12 @@ def write_mds(iterator: Iterable):
         else:
```

Comment:
could you add a short comment or docstring under the `write_mds` function describing what it's doing and input args? Would help clarify things

Reply:
Added.